### PR TITLE
fix: correctly report camera position, fix ECS bug

### PR DESCRIPTION
**Pull request by menduz**
Better with https://github.com/decentraland/unity-client/pull/473

### CODE REVIEW

File: packages/unity-interface/dcl.ts [hunk 1]
```diff
@@ -47,17 +47,19 @@ const preloadedScenes = new Set<string>()
...
+  rotation: Vector3.Zero(),
+  playerHeight: playerConfigurations.height
 }
 
 /////////////////////////////////// HANDLERS ///////////////////////////////////
 
 const browserInterface = {
   /** Triggered when the camera moves */
-  ReportPosition(data: { position: ReadOnlyVector3; rotation: ReadOnlyQuaternion }) {
+  ReportPosition(data: { position: ReadOnlyVector3; rotation: ReadOnlyQuaternion; playerHeight?: number }) {
```
Comment (eordano) [inline packages/unity-interface/dcl.ts:58]: Is this likely to change? I'd try to send less data here. This also looks like something that should go in `config` and set only once.

**Review by eordano — CHANGES_REQUESTED** [review]

File: packages/unity-interface/dcl.ts (hunk 1 above)
Comment (menduz) [reply packages/unity-interface/dcl.ts:58]: may change with the avatar in runtime

Comment (eordano) [reply packages/unity-interface/dcl.ts:58]: This is definitely not something that I would send on *every* position update. Please move that to a engine-explorer initialization parameter (or maybe even a configuration value, for the time being).

**Comment by eordano** [comment]
Merging because #473 is merged

**Pull request merged**